tests/sass/single-line-per-selector.sass

```diff
@@ -19,6 +19,11 @@
   content: 'foo'
 
 
+.foo
+.boo
+  content: 'foo'
+
+
 .foo > a,.bar
```

Comment:
Yes, two selectors on the same line still gives a warning, that has not changed in this pull request.
What is new is that the selector on line 22 that is split over two lines now gives a warning.